stable/heartbeat/values.yaml

```diff
@@ -0,0 +1,64 @@
+image:
+  repository: docker.elastic.co/beats/heartbeat
+  tag: 6.2.4
+  pullPolicy: IfNotPresent
+
+config:
+  # See https://www.elastic.co/guide/en/beats/heartbeat/current/heartbeat-reference-yml.html for reference
+  heartbeat.monitors:
+  - type: icmp
+    schedule: '*/5 * * * * * *'
+    hosts: ["localhost"]
+    ipv4: true
+    timeout: 16s
+    wait: 1s
+
+  processors:
+  - add_cloud_metadata:
+
+  output.file:
```

Comment:
How about leaving an elasticsearch output section commented out just after this one? That would hlep the user configuring the most common use case. Something like this:

```
# output.elasticsearch:
#  hosts: ["localhost:9200"]
#  protocol: "https"
#  username: "elastic"
#  password: "changeme"

Reply:
Sounds good!